.github/workflows/version_checking.yml

```diff
@@ -0,0 +1,39 @@
+name: version_checking
```

Comment:
It shouldn't be a workflow, see tarantool/tarantool actions
https://github.com/tarantool/tarantool/tree/master/.github/actions/environment